### ISSUE TITLE
Show fees and total on Send STX form

### ISSUE THREAD

**Issue by jasperjansz** [blockstack/stacks-wallet#183]
https://www.figma.com/file/VcXbxT1EwXvE1nUlOiBEES/Wallet?node-id=442%3A1515

**Comment by markmhx** [comment]
I believe this is integrated already given the build I've tested

**Comment by markmhx** [comment]
I'm noticing now that we show the fee and total in the preview modal but not previously in the send form.

**Comment by kyranjamie** [comment]
Indeed, there were changes made after I'd implemented the initial form.
